sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -1340,6 +1340,13 @@ object RapidsConf {
     .booleanConf
     .createWithDefault(true)
 
+  val BATCHED_BOUNDED_ROW_WINDOW_MAX_EXTENT: ConfEntryWithDefault[Integer] =
+    conf("spark.rapids.sql.window.batched.bounded.row.extent")
+      .doc("Max value for bounded row window preceding/following extents " +
```

Comment:
nit: also while we are at it could we make it clear that this is for preceding/following independent of each other so we could see up to 2x the number of rows in a window.

Reply:
Should we break this out into two different tunables?